app/services/irs_attempts_api/attempt_event.rb

```diff
@@ -70,7 +70,7 @@ def event_data
           'subject_type' => 'session',
           'session_id' => session_id,
         },
-        'occurred_at' => occurred_at.to_i,
+        'occurred_at_ms' => occurred_at_ms.to_f,
```

Comment:
I believe what they want is more like what JS gives, which is the number if milliseconds since the epoch, so more like:
```suggestion
        'occurred_at_ms' => (occurred_at_ms.to_f * 1000).round,
```

Reply:
It was decided to leave this "as is".